tutorials/House_Prices_Regression_Interpret.ipynb

```diff
@@ -63,7 +63,48 @@
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
nit: Can we revert the change in these lines and 474-487? This will avoid including your local machine path in the notebook. We will address the deprecation warnings separately.

Reply:
makes sense. i'll clean those lines up